examples/Lotka_Volterra_CLI/Lotka_Volterra_PEtab.yml

```diff
@@ -0,0 +1,47 @@
+odes:
+    - stateId: x_1
+      rightHandSide: alpha * x_1 - beta * x_1 * x_2
+      initialValue: 2
+
+    - stateId: x_2
+      rightHandSide: delta * x_1 * x_2 - gamma * x_2
+      initialValue: 2
+
+parameters: 
+   - parameterId: alpha
```

Comment:
I am not completely sure, but 85% of me say no, since the format checking (that is done when importing the model) should complain about non-PEtab entries.